request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -300,6 +310,16 @@ def response_change(self, request, obj):
                 )
         return super().response_change(request, obj)
 
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        # Save the application attributes
+        if obj.application.status == ApplicationStatus.NEW.value:
+            obj.application.status = ApplicationStatus.IN_PROGRESS.value
```

Comment:
What happens if in this/similar places you don't use ".value" at the end?

Reply:
Although it resolves to the same, I have now changed the condition to be similar with the other places by removing the .value part.